Modules/_stat.c

```diff
@@ -11,6 +11,8 @@
  *
  */
 
+#define Py_LIMITED_API 0x030d0000
```

Comment:
Would it make sense to simply use the value of `Py_VERSION_HEX` here?

Reply:
Good point. I guess it is ok to target 3.13 :)

Reply:
This question can be revisited later. IMO Python 3.13 is good for a starting point.

Reply:
Can it use an older version now, or the code uses some new functions added in 3.13?

Reply:
Using ``#define Py_LIMITED_API 0x03050000`` (limited C API of Python 3.5), the ``_stat_`` extension builds successfully.

``static PyModuleDef_Slot stat_slots[]`` fails to build on limited C API older than Python 3.5.

Do you prefer to target the oldest possible version?

Reply:
I think it may be more useful. We can bump it up if start to use newer API.